src/genie/libs/parser/iosxe/show_eigrp.py

```diff
@@ -805,4 +877,132 @@ def cli(self, output=None):
                 int_dict['pend_routes'] = int(group['pend_routes'])
                 continue
 
-        return parsed_dict
\ No newline at end of file
+            m = p5.match(line)
+            if m:
+                group = m.groupdict()
+                int_dict.update({'hello_interval': int(group['hello_interval'])})
+                int_dict.update({'hold_time': int(group['hold_time'])})
+                continue
+
+            m = p6.match(line)
```

Comment:
Please add comment for device output that `p6` will match. Ex:
```
# Split-horizon is enabled
m = p6.match(line)
if m:
    group = m.groupdict()
    if group['state'] == 'enabled':
        split_horizon_state = True
    else:
        split_horizon_state = False
    int_dict.update({'split_horizon_enabled': split_horizon_state})
    continue
```

Please add in similar comments above all other `m = pX.match(line)` lines